packages/bezier-codemod/src/cli.tsx

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
```

Comment:
- https://stackoverflow.com/questions/33509816/what-exactly-does-usr-bin-env-node-do-at-the-beginning-of-node-files

> This line should be added to all files that will be executed through the command line. It's called a [Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)), and basically, it specifies what interpreter the file should be passed to for execution in Unix-like systems.